src/eos-updater-apply.c

```diff
@@ -78,7 +78,6 @@ apply (GTask *task,
   GMainContext *task_context = g_main_context_new ();
   const gchar *update_id = eos_updater_get_update_id (updater);
   const gchar *update_refspec = eos_updater_get_update_refspec (updater);
-  const gchar *orig_refspec = eos_updater_get_original_refspec (updater);
```

Comment:
Yeah that was my review I think :-)
@dbnicholson convinced me that this property could be useful for other clients, so I'd rather leave it in.